src/plugins/dataflow/components/dataflow-program.tsx

```diff
@@ -119,6 +124,7 @@ export class DataflowProgram extends BaseComponent<IProps, IState> {
     this.state = {
       editorContainerWidth: 0,
       lastIntervalDuration: 0,
+      startTimeActual: 0,
```

Comment:
I don't think this needs to be state because I don't think we want it to trigger a re-render and because we don't want changing it to be asynchronous, so it should just be a regular class property.